posthog/api/action.py

```diff
@@ -122,6 +124,12 @@ def create(self, validated_data: Any) -> Any:
         return instance
 
     def update(self, instance: Any, validated_data: dict[str, Any]) -> Any:
+        if validated_data.get("deleted"):
+            if instance.plugin_configs.count():
+                raise serializers.ValidationError(
+                    "Actions with plugins cannot be deleted. Remove it or disable the plugin first."
```

Comment:
looks like it needs to be removed, not just disabled - as the query below would give you all plugin configs
```suggestion
                    "Actions with plugins cannot be deleted. Remove the plugin first."
```